modules/CmpdRegBulkLoader/src/client/CmpdRegBulkLoader.coffee

```diff
@@ -951,12 +951,19 @@ class FileRowSummaryController extends Backbone.View
 			mm = '0' + mm
 		today = '_' + mm + '_' + dd + '_' + yyyy
 
-		# Rename the SDF file representing the current
-		currentFileName = originalFileName.replace "\.sdf", today + "_current_state.sdf"
-    
+		#In some cases, there are artifact .sdf extensions in a file name that cause downloading issues when renanubg a file to .zip
+		#we need to replace ONLY the last .sdf in the file with ".zip" or "{current_date}_current_state.sdf", so we can't use replace which will replace the first
+		fileNameLength = fileName.length #we have to assign this first since for some reason simply using fileName.length - 4 just wont work in line below? 
+		fileNameBase = fileName.slice 0, (fileNameLength)-4 
+		#do the same logic for originalFileName
+		originalFileNameLength = originalFileName.length
+		originalFileName =  originalFileName.slice 0, (originalFileName)-4 
```

Comment:
This should be `originalFileNameBase` right? I think you're clobbering `originalFileName` that we still use below.

Once you have `originalFileNameBase` then you can easily assemble the report displayName:
```
reportName = fileNameBase + ".zip"
reportDisplayName = originalFileNameBase + ".zip"
```